packages/admin-stories/src/admin/helpers/PrettyBytes.tsx

```diff
@@ -0,0 +1,101 @@
+import { PrettyBytes } from "@comet/admin";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+const Story = () => {
```

Comment:
This story and the documentation are identical. @manuelblum is there a way to use a story for both development and the documentation?

Reply:
Theoretically, there is a way (https://storybook.js.org/docs/react/api/mdx#embedding-stories). But I've never managed to make it work correctly. The story is displayed, however it's not possible to use the "Show Code" feature when embedding it like this. It always says "No Code".

Reply:
see https://github.com/vivid-planet/comet-admin/pull/411 and talk to @manuelblum

Reply:
I think it actually can make sense to have two (not identical) stories:

- development story: tests all possible combinations (like you currently do)
- doc stories: no reason to demonstrate the whole range from kB to PB, exactly one typical example (using for example MB) is enough.

Reply:
I shortened the examples in the docs, but added a "Complete Example" section at the end where I embedded the development story. Is that okay?

Reply:
please remove the development story from the docs :)

the plan is to move those into a development section, see https://github.com/vivid-planet/comet-admin/pull/411#issuecomment-854716187 for now it's ok to have them in a stories section.

Reply:
@nsams Done